spec/features/multiple_emails/add_email_spec.rb

```diff
@@ -111,6 +111,7 @@
     user = create(:user, :signed_up)
     sign_in_and_2fa_user(user)
 
+    expect(page).to_not have_link('+ '+ t('account.index.email_add'))
```

Comment:
nit: fix the spacing here please
```suggestion
    expect(page).to_not have_link('+ ' + t('account.index.email_add'))
```